### PR TITLE
Java: update videos for Maven extension

### PR DESCRIPTION
**Pull request by Eskibear**
Signed-off-by: Yan Zhang <yanzh@microsoft.com>

### CODE REVIEW

**Comment by Eskibear** [comment]
cc @nickzhums

File: docs/java/java-build.md [hunk 1]
```diff
@@ -108,16 +108,22 @@ To debug Maven goals, right-click on a goal and start debugging. The Maven exten
 
 Another handy feature provided by this extension is to generate a Maven project from [Archetype](https://maven.apache.org/guides/introduction/introduction-to-archetypes.html). The extension loads archetypes listed in local/remote catalogs. After selection, the extension sends `mvn archetype:generate -D...` to the terminal.
 
-There are two ways to generate a Maven project:
+There are several entries to create a Maven project:
```
Comment (nickzhums) [inline docs/java/java-build.md:111]: several ways?

**Comment by nickzhums** [comment]
everything else looks good

File: docs/java/java-build.md (hunk 1 above)
Comment (nickzhums) [inline docs/java/java-build.md:111]:
```suggestion
There are several ways to create a Maven project:
```

**Review by nickzhums — APPROVED** [review]

**Review by nickzhums — APPROVED** [review]

**Pull request merged**